src/openforms/registrations/contrib/stuf_zds/plugin.py

```diff
@@ -37,16 +37,19 @@ class ZaakOptionsSerializer(JsonSchemaSerializerMixin, serializers.Serializer):
         help_text=_("Zaaktype code for newly created Zaken in StUF-ZDS"),
     )
     zds_zaaktype_omschrijving = serializers.CharField(
-        required=True,
+        required=False,
+        allow_null=True,
         help_text=_("Zaaktype description for newly created Zaken in StUF-ZDS"),
     )
 
     zds_zaaktype_status_code = serializers.CharField(
-        required=True,
+        required=False,
+        allow_null=True,
         help_text=_("Zaaktype status code for newly created zaken in StUF-ZDS"),
     )
     zds_zaaktype_status_omschrijving = serializers.CharField(
-        required=True,
+        required=False,
+        allow_null=True,
```

Comment:
there is no reason to set `allow_null=True` - string-based inputs should only have one type of empty value: the empty string.

You need to update the test to use empty strings instead to reflect actual user input.

Reply:
@sergei-maertens The API sends null values in the payload:

![image](https://user-images.githubusercontent.com/58147939/177142576-0393c1ad-0958-458a-8b37-feb1be52f1f4.png)

Reply:
it should be empty strings instead of null